src/index.js

```diff
@@ -3,12 +3,16 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { GlobalValues } from './ValueProvider';
```

Comment:
It is actually more common to keep the context wrapped in App.js but it can stay here